src/app/shared/dialog-box/dialog-box.component.scss

```diff
@@ -0,0 +1,18 @@
+textarea {
+  width: 80% !important;
```

Comment:
why are we overriding all the text area width to 80% ?

Reply:
Also it seems that we don't have this component so we can create some scss for it that we load in global.scss
We will have to remove all the hardcoded values, we can use the ones provided in the styleguide for the scaffolding elements such as color, font-size, font-weight.
If we don't have them we can create variables for them.